parser/openfoodfacts_taxonomy_parser/parser/taxonomy_parser.py

```diff
@@ -226,14 +245,26 @@ def _harvest_entries(self, filename: str, entries_start_line: int) -> Iterator[N
                     self.parser_logger.error(msg)
                 else:
                     data = self._remove_separating_line(data)
+                    if data.get_node_type() == NodeType.ENTRY:
```

Comment:
why only for NodeType.ENTRY ?

We also need to keep comments for stopwords and synonyms

Reply:
For stopwords and synonyms, comments are stored in the `preceding_lines` property, as before

Reply:
ok